tsconfig.eslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  // Reset exclusions in order to include files with .test extension.
```

Comment:
Comments are not permitted in JSON

Reply:
This is allowed for TypeScript JSON files, see: https://github.com/microsoft/TypeScript/pull/5450